src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -47,6 +47,7 @@ internal class HttpApiHandler : IDisposable
         private const string ShowHistoryOutputParameter = "showHistoryOutput";
         private const string ShowInputParameter = "showInput";
         private const string FetchStateParameter = "fetchState";
+        private const string InstanceIdStartsWithParameter = "instanceIdStartsWith";

```

Comment:
In the SDKs, we call this "instance ID *prefix*". Any reason not to be consistent with this query string name?

Reply:
The "startswith" is a better name IMO, as it is more self-explanatory and is commonly used in other circumstances (e.g. Kusto and C#). 

Then again, consistency inside the codebase is also worth something.

I am o.k. with either.